drltrace/drltrace_frontend.cpp

```diff
@@ -97,6 +97,12 @@ static droption_t<bool> op_print_ret_addr
 (DROPTION_SCOPE_CLIENT, "print_ret_addr", false, "Print library call's return address",
  "Print return addresses of library calls.");
 
+static droption_t<uint> op_all_args
+(DROPTION_SCOPE_CLIENT, "all_args", 2, "Number of library call arguments to print",
+ "Number of library call arguments to print. Specify 0 to disable. Drltrace will print "
+ "symbolic representation of arguments in case of known library call including all "
+ "known arguments.");
```

Comment:
"-num_unknown_args 3 -max_args 7" means *never* print more than 7, and furthermore don't print more than 3 when the types and # of args are not known.

Definitely there is a use case where no args should be printed.  The question is whether some users would want to limit the # of args to some small number for cleaner output: see some but not all.  On Linux there are never more than 6, but on Windows where there can be over 20 it may be a different story.